src/dialogs/more-info/ha-more-info-device-entities-shortcuts.ts

```diff
@@ -0,0 +1,156 @@
+import { HassEntity } from "home-assistant-js-websocket";
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { styleMap } from "lit/directives/style-map";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../common/dom/fire_event";
+import { computeStateDisplay } from "../../common/entity/compute_state_display";
+import { stateActive } from "../../common/entity/state_active";
+import { stateColor } from "../../common/entity/state_color";
+import { stateIconPath } from "../../common/entity/state_icon_path";
+import "../../components/ha-chip";
+import "../../components/ha-chip-set";
+import "../../components/ha-icon";
+import "../../components/ha-svg-icon";
+import { HomeAssistant } from "../../types";
+
+declare global {
+  interface HASSDomEvents {
+    "shortcut-clicked": { entityId: string };
+  }
+}
+
+@customElement("ha-more-info-device-entities-shortcuts")
+class MoreInfoDevicesEntitiesShortcuts extends LitElement {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: HassEntity;
+
+  private _deviceEntityEntries = memoizeOne((deviceId: string) => {
+    if (!deviceId) return [];
+
+    return Object.values(this.hass!.entities).filter(
+      (entity) =>
+        entity.device_id === deviceId &&
+        !entity.hidden_by &&
+        !entity.disabled_by &&
+        !entity.entity_category
+    );
+  });
+
+  private _handleChipClick(ev) {
+    if (ev.defaultPrevented) {
+      return;
+    }
+    if (ev.type === "keydown" && ev.key !== "Enter" && ev.key !== " ") {
+      return;
+    }
+    const entityId = (ev.target as any).entityId as string;
+    fireEvent(this, "shortcut-clicked", { entityId });
```

Comment:
I think it is better to just do:
```
fireEvent(this, "hass-more-info", { entityId });
```
As it will also handle things like back navigation

Reply:
This creates a flickering effect, that why I directly set the entity.